Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM quay.io/ukhomeofficedigital/alpine:v3.13
+
+ENV USER user-hocs-bulk-case-action
+ENV USER_ID 1000
+ENV GROUP group-hocs-bulk-case-action
+ENV NAME hocs-bulk-case-action
+ENV JAR_PATH build/libs
+
+ENV workflow-address http://host.docker.internal:8091
+ENV x-auth-groups /COMP_CASEADMIN_bdhvyvg
```

Comment:
These will be passed through secrets in K8, so would be worth having these in application properties so they can be overidden.

Reply:
I've removed these from the Dockerfile so they can be set in K8